web-assembly/hosted/okta-hosted-login/okta-blazor-web-assembly-example/Pages/FetchData.razor

```diff
@@ -0,0 +1,50 @@
+@page "/fetchdata"
+@using Microsoft.AspNetCore.Authorization
+@using Microsoft.AspNetCore.Components.WebAssembly.Authentication
+@using Microsoft.Extensions.Configuration
+@attribute [Authorize]
+@inject IHttpClientFactory ClientFactory
+@inject IConfiguration Configuration
+
+<h1>Server speaks</h1>
+
+<p>This component demonstrates fetching data from the server.</p>
+
+@if (serverMessages == null)
+{
+<p><em>Loading...</em></p> }
+else
+{
+<table class="table">
+    <thead>
+        <tr>
+            <th>Date</th>
+            <th>Text</th>
+        </tr>
+    </thead>
+    <tbody>
+        @foreach (var message in serverMessages.Messages)
+        {
+<tr>
+    <td>@message.Date.ToShortDateString()</td>
+    <td>@message.Text</td>
+</tr>}
+    </tbody>
+</table>}
+
+@code { 
+    private ServerMessages serverMessages;
+
+    protected override async Task OnInitializedAsync()
+    {
+        var httpClient = ClientFactory.CreateClient("BlazorClient.ServerApi");

```

Comment:
Can you remind me where you configure the client to send the access token?

Reply:
Client is configured in Program.cs